lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Transactions.hs

```diff
@@ -1349,7 +1328,7 @@ spec = describe "SHELLEY_TRANSACTIONS" $ do
                     Nothing
             rl <- request @([ApiTransaction n]) ctx linkList Default Empty
             verify rl [expectListSize 2]
-            pure (getFromResponse Prelude.id rl)
+            pure (getFromResponse idFunc rl)
```

Comment:
Yes, `id` is a common and useful variable name, and not just for record fields.

You still have the `id` function in scope, and `idFunc` is an additional name which is shorter than `Cardano.Wallet.Prelude.id`.

Enabling `NoFieldSelectors` would cause rather more inconvience than simply aliasing a function, and it's not available in the GHC version we are using anyway.